src/lib.rs

```diff
@@ -220,6 +227,20 @@ type PreviousRlpEncodedValues = Vec<Option<Bytes>>;
 
 const MAX_ENR_SIZE: usize = 300;
 
+// Constants used for fields
+const ID_ENR_KEY: &str = "id";
+const ENR_VERSION: &str = "v4";
+pub const IP_ENR_KEY: &str = "ip";
+pub const IP6_ENR_KEY: &str = "ip6";
+pub const TCP_ENR_KEY: &str = "tcp";
+pub const TCP6_ENR_KEY: &str = "tcp6";
+pub const UDP_ENR_KEY: &str = "udp";
+pub const UDP6_ENR_KEY: &str = "udp6";
+#[cfg(feature = "quic")]
+pub const QUIC_ENR_KEY: &str = "quic";
+#[cfg(feature = "quic")]
+pub const QUIC6_ENR_KEY: &str = "quic6";
```

Comment:
you only ever use these as bytes. I think it's easier to go
```rust
pub const QUIC6_ENR_KEY: &[u8: 5] = "quic6";
```
and you can save yourself all those `as_bytes` and similar, and needing to convert from key to string to check the spec reserved keys